README.md

```diff
@@ -77,13 +67,13 @@ const orderSchema = require('./order-schema');
 
 // merge conflicting "Query", "Mutation", and "Subscription" definitions
 const typeDefs = mergeTypeDefs([
-  ...customerSchema.types,
-  ...orderSchema.types
+  customerSchema.typeDefs,
+  orderSchema.typeDefs,
```

Comment:
You can still do an array of typeDefs since it's based on the developers implantation and it doesn't affect this library. I just changed it because it seemed to read a little cleaner and had less variables. I can always do `customerSchema` one way and `orderSchema` the other